css/styles.css

```diff
@@ -0,0 +1,287 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  overflow-x: hidden;
+  margin-bottom: -236px;
+}
+
+.hidden {
+  visibility: hidden;
+  position: absolute;
+}
+
+.nav-container {
+  background-image: linear-gradient(to bottom, #626262 50%, #797979 50%);
+  width: 985px;
+  height: 37px;
+  border-radius: 8px;
+  border: 1px solid #5e5e5e;
+  position: relative;
+  top: 42px;
+  margin: auto;
+  z-index: 5;
+}
+
+#cards-links li {
+  border: 1px solid white;
+  list-style-type: none;
+  width: 25%;
+}
+
+#right-bottom-links li {
+  list-style-type: none;
+  margin-right: 3px;
+  margin-left: 3px;
+  padding-left: 3px;
+  padding-right: 5px;
+}
+
+#copyright li {
+  list-style-type: none;
+  margin-right: 3px;
+  margin-left: 3px;
+  padding-left: 3px;
+  padding-right: 5px;
+}
+
+#nav-links {
+  background-image: url("/images/nav-surround.png");
```

Comment:
The hero image is not showing, try this instead `url("../images/nav-surround.png")`